config/webpacker.yml

```
@@ -96,3 +96,6 @@ production:
 
   # Cache manifest.json for performance
   cache_manifest: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)